docs/en/operation/Metrics-System.md

```diff
@@ -96,6 +97,43 @@ The filename will correspond with the metric name.
 
 Refer to `metrics.properties.template` for all possible sink specific configurations. 
 
+### Prometheus Sink Setup
+
+[Prometheus](https://prometheus.io/) is a monitoring tool that can help monitoring Alluxio metrics changes.
+
+In the metrics property file, `$ALLUXIO_HOME/conf/metrics.properties` by default, add the following properties:
+
+```
+# Enable PrometheusMetricsServlet
+sink.prometheus.class=alluxio.metrics.sink.PrometheusMetricsServlet
+```
+
+You can send an HTTP request to `/metrics/prometheus/` of the Alluxio leading master or workers to get a snapshot of metrics in Prometheus format. 
```

Comment:
a HTTP request to the `/metrics/prometheus/` endpoint of the Alluxio leading master or workers

Reply:
Got it! thanks